src/Card/Card.component.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react"
+import styled, { css } from "styled-components"

```

Comment:
Styles should live on their own file. Basically there should be a `Card.styles.ts` with all the `styled-components` and those should be imported into `Card.component.tsx`